charts/hoprd-operator/templates/persistence-volume-claim.yaml

```diff
@@ -2,22 +2,22 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   labels:
-    {{- include "hopr-operator.labels" . | nindent 4 }}
-    {{- if .Values.commonLabels }}
-    {{- .Values.commonLabels | toYaml | nindent 4 }}
+    {{- include "hoprd-operator.labels" . | nindent 4 }}
+    {{- if .Values.operator.commonLabels }}
+    {{- .Values.operator.commonLabels | toYaml | nindent 4 }}
     {{- end }}
-{{- if .Values.commonAnnotations }}
+{{- if .Values.operator.commonAnnotations }}
   annotations:
-    {{- .Values.commonAnnotations | toYaml | nindent 4 }}
+    {{- .Values.operator.commonAnnotations | toYaml | nindent 4 }}
 {{- end }}
-  name: {{ include "hopr-operator.fullname" . }}
+  name: {{ include "hoprd-operator.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
 spec:
   accessModes:
     - ReadWriteOnce
     - ReadWriteMany
```

Comment:
I used readWriteMany because I the operator and the jobs it launches share the same persistence volume. I think we can change it, but will require some time and more testing.

Reply:
I've been able to take off the accessMode.